### PR TITLE
docs(en-US): add user on AC page

### PR DESCRIPTION
**Pull request by IceHe**
Add user on Admin Console page

image

### CODE REVIEW

File: docs/docs/recipes/manage-users/on-admin-console-page.md [hunk 1]
```diff
@@ -7,7 +7,18 @@ sidebar_position: 1
 
 ## Add user
```
Comment (simeng-li) [inline docs/docs/recipes/manage-users/on-admin-console-page.md:8]: better make them as an ordered or unordered bullet list?

File: docs/docs/recipes/manage-users/on-admin-console-page.md [hunk 2]
```diff
@@ -7,7 +7,18 @@ sidebar_position: 1
...
 
-TODO: admin manually add
+Not only can the end-user sign up by his/herself, but also the admin users can manually add accounts for them.
```
Comment (simeng-li) [inline docs/docs/recipes/manage-users/on-admin-console-page.md:10]: Sth like, `Admin users could create a new account through Admin Console in the name of an end-user. ` or simply `Admin users could create an account for a new user upon their request through Admin Console.`

Comment (simeng-li) [reply docs/docs/recipes/manage-users/on-admin-console-page.md:10]: him/herself BTW...

File: docs/docs/recipes/manage-users/on-admin-console-page.md [hunk 3]
```diff
@@ -7,7 +7,18 @@ sidebar_position: 1
...
+
+1. In the "User Management" panel, click the "+ Add User" button at the top right corner.
```
Comment (simeng-li) [inline docs/docs/recipes/manage-users/on-admin-console-page.md:12]: nit: feels like, `on the pane` is more comfortable, not sure, just a feeling.

File: docs/docs/recipes/manage-users/on-admin-console-page.md [hunk 4]
```diff
@@ -7,7 +7,18 @@ sidebar_position: 1
...
+2. In the "Add User" dialog, fill out the "Username" (for sign-in) and "Full name" input boxes, and click the "Add User" button to confirm.
+3. After successfully adding a new user, you can see their username and initial password in the dialog.
+   Click the eye icon following the password to show or hide the actual password.
+4. Click the "Copy" button to copy the **username** and **initial password** to the clipboard so that you can send it to the user to sign in.
```
Comment (simeng-li) [inline docs/docs/recipes/manage-users/on-admin-console-page.md:16]: Too many details?  We will have multiple screenshots for different steps.

**Review by wangsijie — APPROVED** [review]
Approve to unblock, Stories with scenario will be better.

File: docs/docs/recipes/manage-users/on-admin-console-page.md [hunk 5]
```diff
@@ -7,7 +7,18 @@ sidebar_position: 1
...
+
+:::caution
+After closing the "Add User" dialog, you can no longer view the actual password.
```
Comment (simeng-li) [inline docs/docs/recipes/manage-users/on-admin-console-page.md:19]: ~~actual~~

File: docs/docs/recipes/manage-users/on-admin-console-page.md (hunk 1 above)
Comment (IceHe) [reply docs/docs/recipes/manage-users/on-admin-console-page.md:8]: Each section seems to be a little longer.
Maybe they're not suitable to be contained in an ordered or unordered bullet list.

We can reorg them after finishing their content later.

File: docs/docs/recipes/manage-users/on-admin-console-page.md (hunk 5 above)
Comment (IceHe) [reply docs/docs/recipes/manage-users/on-admin-console-page.md:19]: improved.

File: docs/docs/recipes/manage-users/on-admin-console-page.md (hunk 3 above)
Comment (IceHe) [reply docs/docs/recipes/manage-users/on-admin-console-page.md:12]: The user may view the the “Add user” section directly, and don't know which "panel".
So I emphasize the "User Management" panel here.

Later we may reorg this step like #46 

image

File: docs/docs/recipes/manage-users/on-admin-console-page.md (hunk 4 above)
Comment (IceHe) [reply docs/docs/recipes/manage-users/on-admin-console-page.md:16]: Will do it later.

File: docs/docs/recipes/manage-users/on-admin-console-page.md (hunk 2 above)
Comment (IceHe) [reply docs/docs/recipes/manage-users/on-admin-console-page.md]: Updated. PTAL.

**Pull request merged**